cmake/toolchains/mt.runtime.win32.toolchain.cmake

```diff
@@ -26,15 +26,19 @@ if(use_static_runtime)
     foreach(lang C CXX)
         foreach(build_type "" "_DEBUG" "_MINSIZEREL" "_RELEASE" "_RELWITHDEBINFO")
             set(flag_var "CMAKE_${lang}_FLAGS${build_type}_INIT")
-            set(flag_var_noninit "CMAKE_${lang}_FLAGS${build_type}")
             string(REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
+        endforeach()
+    endforeach()
+    foreach(type EXE SHARED MODULE)
+        foreach(build_type "" "_DEBUG" "_MINSIZEREL" "_RELEASE" "_RELWITHDEBINFO")
+            set(flag_var "CMAKE_${type}_LINKER_FLAGS${build_type}_INIT")
             if (build_type STREQUAL "_DEBUG")
-                set(${flag_var_noninit} "${${flag_var_noninit}} /MTd")
+                string(APPEND ${flag_var} " /NODEFAULTLIB:libucrtd.lib /DEFAULTLIB:ucrtd.lib")
```

Comment:
Are we linking components with deferent versions of CRT? (linker #pragmas)
Please document why is that needed.

Reply:
Lining with /verbose:lib might help